__tests__/app.test.js

```diff
@@ -73,8 +73,32 @@ describe("/api/articles", () => {
         .get("/api/articles")
         .expect(200)
         .then((response) => {
-            expect(response.body.articles).toBeSortedBy('created_at', { descending: true })
-        })
+            expect(response.body.articles).toBeSortedBy('created_at', { descending: true });
+        });
+    })
+    test("GET: 200 responds with articles when passed valid column and order queries in correct form", () => {
+        return request(app)
+        .get("/api/articles?sort_by=title&order=asc")
+        .expect(200)
+        .then((response) => {
+            expect(response.body.articles).toBeSortedBy('title', { ascending: true });
```

Comment:
`toBeSortedBy` doesn't have an `ascending` option: https://github.com/JonanLowe/nc-news-jl/pull/11#pullrequestreview-2374592125
Instead it defaults to ascending.